docs/bounties.md

```diff
@@ -1,7 +1,10 @@
 ## Bounties
 
-We use bounties to incentivize development. To receive a bounty, you agree to these conditions:
+We use bounties to incentivize development and reward contributors. All issues available for a bounty are listed [on the Kanban board](https://github.com/orgs/haveno-dex/projects/2). It's possible to list on each repository the issues with a bounty on them, by searching issues with the '💰bounty' label.
 
+To receive a bounty, you agree to these conditions:
+
+- Your changes must follow the [styling guidelines](contributing.md).
```

Comment:
Is this link correct since it's in ./docs?

Reply:
This file is in `/docs` too, it's a relative link. I guess could be changed to `/docs/CONTRIBUTING.md` if wanted (the capitalization needs to be fixed regardless).